cognite/client/_cognite_client.py

```diff
@@ -48,6 +49,7 @@ def __init__(
         headers: Dict[str, str] = None,
         timeout: int = None,
         token: Union[str, Callable[[], str], None] = None,
+        disable_pypi_version_check: bool = None,
```

Comment:
```suggestion
        disable_pypi_version_check: bool = False,
```

Reply:
I agree thats a bit clunky, but then how we detect no argument so we can fall back to env vars?

Reply:
Ah right. Set type hint to Optional[bool] in that case.